pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -560,6 +561,11 @@ type CloudConfig struct {
 		//yourself in an non-AWS cloud and open an issue, please indicate that in the
 		//issue body.
 		DisableStrictZoneCheck bool
+
+		//The aws provider will lookup docker-login credentials for ECR in each region for
+		//the current AWS account. To lookup credentials for cross-account ECR access, supply
+		//the additional registry IDs.
+		EcrRegistryId []string
```

Comment:
Does this replace or add to the existing (default) policy?

Reply:
(I think it's fine either way, because I think the answer is that if you want to keep the default account you specify `*` as one of the strings, but I think that should be explicit somewhere.)

Reply:
If present it will replace the default in its current form - I figured it may be useful to not attempt to get a credentialProvider for the current account in some scenarios. I can add to that comment though.